chart/values.yaml

```diff
@@ -21,6 +21,11 @@ initImage:
   repository: busybox
   # initImage.repository -- Tag to use for initContainers docker image
   tag: latest
+  # initImage.pullPolicy -- Kubernetes image pullPolicy to use for fusionauth-app
+  pullPolicy: Always
+
+
+  # initImage
```

Comment:
Can you fix this dangling comment and the extra blank line?

Reply:
Done.